### PR TITLE
Removed Stop Button from SO and PO

### PR DESCRIPTION
**Pull request by patilsangram**
Hi, 
As per your suggestion in PR - https://github.com/frappe/erpnext/pull/4836#
I removed Stop Button instead of Close from SO and PO

Thanks,
Sangram
New Indictrans Technologies Pvt Ltd

### CODE REVIEW

File: erpnext/accounts/report/ordered_items_to_be_billed/ordered_items_to_be_billed.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
...
...`name` as \"Sales Order:Link/Sales Order:120\",\n `tabSales Order`.`customer` as \"Customer:Link/Customer:120\",\n `tabSales Order`.`status` as \"Status\",\n `tabSales Order`.`transaction_date` as \"Date:Date\",\n `tabSales Order`.`project_name` as \"Project\",\n `tabSales Order Item`.item_code as \"Item:Link/Item:120\",\n `tabSales Order Item`.base_amount as \"Amount:Currency:110\",\n (`tabSales Order Item`.billed_amt * ifnull(`tabSales Order`.conversion_rate, 1)) as \"Billed Amount:Currency:110\",\n (ifnull(`tabSales Order Item`.base_amount, 0) - (ifnull(`tabSales Order Item`.billed_amt, 0) * ifnull(`tabSales Order`.conversion_rate, 1))) as \"Pending Amount:Currency:120\",\n `tabSales Order Item`.item_name as \"Item Name::150\",\n `tabSales Order Item`.description as \"Description::200\",\n `tabSales Order`.`company` as \"Company:Link/Company:\"\nfrom\n `tabSales Order`, `tabSales Order Item`\nwhere\n `tabSales Order Item`.`parent` = `tabSales Order`.`name`\n and `tabSales Order`.docstatus = 1\n and `tabSales Order`.status not in (\"Closed\")\n and ifnull(`tabSales Order Item`.billed_amt,0) < ifnull(`tabSales Order Item`.amount,0)\norder by `tabSales Order`.transaction_date asc",
```
Comment (nabinhait) [inline erpnext/accounts/report/ordered_items_to_be_billed/ordered_items_to_be_billed.json]: status != "Closed"

File: erpnext/accounts/report/purchase_order_items_to_be_billed/purchase_order_items_to_be_billed.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
...
...e:Date:100\",\n\t`tabPurchase Order`.`supplier` as \"Supplier:Link/Supplier:120\",\n\t`tabPurchase Order Item`.`project_name` as \"Project\",\n\t`tabPurchase Order Item`.item_code as \"Item Code:Link/Item:120\",\n\t`tabPurchase Order Item`.base_amount as \"Amount:Currency:100\",\n\t(`tabPurchase Order Item`.billed_amt * ifnull(`tabPurchase Order`.conversion_rate, 1)) as \"Billed Amount:Currency:100\", \n\t(`tabPurchase Order Item`.base_amount - (ifnull(`tabPurchase Order Item`.billed_amt, 0) * ifnull(`tabPurchase Order`.conversion_rate, 1))) as \"Amount to Bill:Currency:100\",\n\t`tabPurchase Order Item`.item_name as \"Item Name::150\",\n\t`tabPurchase Order Item`.description as \"Description::200\",\n\t`tabPurchase Order`.company as \"Company:Link/Company:\"\nfrom\n\t`tabPurchase Order`, `tabPurchase Order Item`\nwhere\n\t`tabPurchase Order Item`.`parent` = `tabPurchase Order`.`name`\n\tand `tabPurchase Order`.docstatus = 1\n\tand `tabPurchase Order`.status not in (\"Closed\")\n\tand (ifnull(`tabPurchase Order Item`.billed_amt, 0) * ifnull(`tabPurchase Order`.conversion_rate, 1)) < ifnull(`tabPurchase Order Item`.base_amount, 0)\norder by `tabPurchase Order`.transaction_date asc",
```
Comment (nabinhait) [inline erpnext/accounts/report/purchase_order_items_to_be_billed/purchase_order_items_to_be_billed.json]: status != "Closed"

File: erpnext/buying/doctype/purchase_order/purchase_order.js [hunk 3]
```diff
@@ -58,13 +58,13 @@ erpnext.buying.PurchaseOrderController = erpnext.buying.BuyingController.extend(
...
-		if(doc.docstatus == 1 && in_list(["Stopped", "Closed", "Delivered"], doc.status)) {
+		if(doc.docstatus == 1 && in_list(["Closed", "Delivered"], doc.status)) {
 			if (this.frm.has_perm("submit")) {
-				cur_frm.add_custom_button(__('Re-open'), this.unstop_purchase_order, __("Status"));
+				cur_frm.add_custom_button(__('Re-open'), this.unclose_purchase_order, __("Status"));
 			}
 		}
 
-		if(doc.docstatus == 1 && !in_list(["Stopped", "Closed"], doc.status)) {
+		if(doc.docstatus == 1 && !in_list(["Closed"], doc.status)) {
```
Comment (nabinhait) [inline erpnext/buying/doctype/purchase_order/purchase_order.js]: doc.status != "Closed"

File: erpnext/selling/doctype/sales_order/sales_order_list.js [hunk 4]
```diff
@@ -2,58 +2,51 @@ frappe.listview_settings['Sales Order'] = {
...
 			listview.call_for_selected_items(method, {"status": "Closed"});
 		});
 
-		listview.page.add_menu_item(__("Stop"), function() {
-			listview.call_for_selected_items(method, {"status": "Stoped"});
-		});
-
 		listview.page.add_menu_item(__("Re-open"), function() {
-			listview.call_for_selected_items(method, {"status": "Unstop"});
+			listview.call_for_selected_items(method, {"status": "Unclose"});
```
Comment (nabinhait) [inline erpnext/selling/doctype/sales_order/sales_order_list.js]: Re-open?

File: erpnext/stock/stock_balance.py [hunk 5]
```diff
@@ -92,7 +92,7 @@ def get_reserved_qty(item_code, warehouse):
 					and parenttype="Sales Order"
 					and item_code != parent_item
 					and exists (select * from `tabSales Order` so
-					where name = dnpi_in.parent and docstatus = 1 and status not in ('Stopped','Closed'))
+					where name = dnpi_in.parent and docstatus = 1 and status not in ('Closed'))
```
Comment (nabinhait) [inline erpnext/stock/stock_balance.py]: status != 'Closed'

File: erpnext/stock/stock_balance.py [hunk 6]
```diff
@@ -102,7 +102,7 @@ def get_reserved_qty(item_code, warehouse):
 				and (so_item.delivered_by_supplier is null or so_item.delivered_by_supplier = 0)
 				and exists(select * from `tabSales Order` so
 					where so.name = so_item.parent and so.docstatus = 1
-					and so.status not in ('Stopped','Closed')))
+					and so.status not in ('Closed')))
```
Comment (nabinhait) [inline erpnext/stock/stock_balance.py]: status != 'Closed'

File: erpnext/selling/doctype/sales_order/sales_order_list.js (hunk 4 above)
Comment (nabinhait) [inline erpnext/selling/doctype/sales_order/sales_order_list.js]: listview.call_for_selected_items(method, {"status": "Submitted"});

**Comment by nabinhait** [comment]
thanks

**Comment by patilsangram** [comment]
Thank you Sir.

**Pull request merged**